src/util/openfaas/index.js

```diff
@@ -64,6 +66,32 @@ const callWithRetry = async (
   }
 };
 
+const reconcileFunction = async (fnName) => {
```

Comment:
Rather than executing the call on the openfaas-gateway, give back the formatted payload to the caller and let the caller make api calls from that payload to gateway api.